tests/check_new_syntax.py

```diff
@@ -12,6 +12,23 @@
 CONTEXT_MANAGER_ALIASES = {"ContextManager": "AbstractContextManager", "AsyncContextManager": "AbstractAsyncContextManager"}
 CONTEXTLIB_ALIAS_ALLOWLIST = frozenset({Path("stdlib/contextlib.pyi"), Path("stdlib/typing_extensions.pyi")})
 
+IMPORTED_FROM_TYPING_NOT_TYPING_EXTENSIONS = frozenset(
+    {"ClassVar", "Type", "NewType", "overload", "Text", "Protocol", "runtime_checkable"}
```

Comment:
```suggestion
    {"ClassVar", "Type", "NewType", "overload", "Text", "Protocol", "runtime_checkable", "NoReturn"}
```
I believe this one is safe too

Reply:
```python
>>> import typing_extensions
>>> "NoReturn" in typing_extensions.__all__
False
>>> typing_extensions.NoReturn
typing.NoReturn
```
What's up with this?

Reply:
https://github.com/python/typing/blob/a2371460d184c96aab7a69acc47fd059f875e3b4/typing_extensions/src/typing_extensions.py#L91

Reply:
Seems like an issue, want to open a PR at python/typing? I guess not many people do `from typing_extensions import *` + it only really affects Python 3.5 to 3.6.2

Reply:
Meh, I kind of don't feel like opening a PR for a Python version that will be officially unsupported from next week :p

It kind of seems like it only affects one person: me, as I was using `__all__` as a handy list of the names were public in `typing_extensions`